R/borel.r

```diff
@@ -27,20 +27,18 @@ dborel <- function(x, mu, log = FALSE) {
 ##' Random numbers are generated by simulating from a Poisson branching process
 ##' @param n Number of random variates to generate.
 ##' @inheritParams dborel
-##' @param infinite A threshold above which the random number is set to
-##'  `Inf` as a form of truncation; A number coercible to integer. This
-##'  argument is important because `rborel()` uses `simulate_summary()`
-##'  internally with `offspring_dist = rpois`, so when `mu >= 1`, the
-##'  simulation could grow out of proportion and chain summaries (the random
-##'  numbers generated here) need to be truncated using
-##'  `infinite`.
+##' @param censor_at A stopping criterion; `<numeric>`. Defaults to `Inf`. A
+##' value above which the simulation ends and the random number is set to
+##' `Inf` (as a form of censoring). `rborel()` uses [simulate_summary()]
+##' internally with a poisson offspring distribution, so if `mu >= 1`, the
+##' simulation could proceed unendingly. This parameter is used to prevent this.
```

Comment:
```suggestion
##' `Inf` (as a form of censoring). `rborel()` simulates chain sizes using [simulate_summary()]
##'  with a Poisson offspring distribution, so if `mu >= 1`, the
##' simulation could proceed unendingly. This parameter is used to prevent this.
```